app/models/bag.rb

```diff
@@ -12,7 +12,7 @@ def data
     file_locations = {}
     @thesis.files.map { |f| file_locations["data/#{f.filename}"] = f.blob }
 
-    # TODO: build csv, add it to hash
+    file_locations['data/metadata.csv'] = ArchivematicaMetadata.new(@thesis).to_csv
```

Comment:
This will need a slight rewrite after rebasing, since we're now using an array of arrays in this method instead of a hash.

Reply:
I don't think that array of arrays work is pushed yet right? So this is a signal to that work to be careful if it moves forward and not for this work to particularly care right?

Reply:
Oh yeah, you're right. My mistake. Too many branches floating around.